src/main/java/org/opensearch/agent/tools/RAGTool.java

```diff
@@ -71,6 +71,11 @@ public class RAGTool implements Tool {
     @Setter
     private Parser outputParser;
 
+    @Override
```

Comment:
This is a new method defined in Tool's interface?

Reply:
Nope, it's an existing method but none of the tools override it. For RAGTool and VectorDBTool, they need original input(plain text) instead of a LLM-transformed input.

Reply:
Can you add some simple comments to describe why original input is needed for this two tools?

Reply:
Does Agent framework support this now?